### ISSUE TITLE
Scripting Functionality

### ISSUE THREAD

**Issue by aaroncohen73** [osuosl/climesync#39]
In addition to interactive mode (Currently the only way to use Climesync), commands should be able to be called from scripts. As mentioned in #38 [docopt](http://docopt.org/) would be an ideal module for this task.

Possible examples of usage and syntax:
```
./climesync.py create-user new_username new_password --display-name="John Doe" --site-admin=true

./climesync.py get-times --users="user1 user2" --start-date=2016-05-01 --end-date=2016-05-10

./climesync.py delete-project prj_slug
```

**Comment by Kennric** [comment]
Also useful, being able to submit time quickly with a commandline, especially in conjunction with a config file containing the url and auth info. For example, i'd like to be able to:
`
climesync --project gwm --activity tests --time 4h `
or: 
`climesync -p gwm -a tests -t 4h`

and have the app submit a time of 4 hours on gwm with today's date, all other options set to defaults (from the app or from my config file).

**Comment by ElijahCaine** [comment]
I like @aaroncohen73's sub-commands. That'll make it pretty robust. Being able to pass arguments will make it very easy to use this and extend with scripts.

**Comment by aaroncohen73** [comment]
Closed via #42
